Source/WTF/wtf/spi/darwin/CommonCryptoSPI.h

```diff
@@ -34,6 +34,11 @@
 #endif
 #endif
 
+#if !PLATFORM(IOS) || USE(APPLE_INTERNAL_SDK)
```

Comment:
You don't have to guard the declarations in the header. It's fine if they're there as long as they aren't used.